### ISSUE TITLE
Show randomized idea on top of ideas list

### ISSUE THREAD

**Issue by marc1706** [phpbb/ideas#72]
Show a random idea in the form of something like a card on top of the ideas list. This would result in any random idea being shown on top of the list to get people to see more than just the top ideas and find ideas they might not know about.

**Comment by VSEphpbb** [comment]
Like this?
![screen shot 2017-09-10 at 3 29 16 pm](https://user-images.githubusercontent.com/303711/30253606-a95947bc-963d-11e7-82c8-597072b03a2c.png)

**Comment by DavidIQ** [comment]
If we're showing 4 for the other lists it might make more sense to show the same amount of random ideas, but yeah that looks good.

**Comment by VSEphpbb** [comment]
(actually it's 5 I just don't have that many ideas on my test) I don't like the idea of 5 (not even crazy about 1). If we were to do 5 then I'd say put it towards the bottom because a group of rando ideas is less meaningful than the top 5 or the 5 newest ones.

**Comment by DavidIQ** [comment]
It might be less meaningful, but it would be drawing attention away from those which are hogging pretty much all the votes, so I wouldn't put it at the bottom. If users never go through the list of ideas they'd just vote for whatever they see on the top or latest ideas lists, which is what's currently happening. Problem with just having one is that if it is really random then chances are high that it will just be something in one of the other lists.

**Comment by marc1706** [comment]
No, not like that. I meant card when I said card. I'll try to mock up something when I'm back home. The card is supposed to show not the same list view of ideas but rather what you see when you click on the idea (more or less at least, can be reduced to just the idea title, description, and current up/down votes).

**Comment by DavidIQ** [comment]
Ah right, sounds good.

**Comment by Dode123** [comment]
> Show a random idea in the form of something like a card on top of the ideas list. This would result in any random idea being shown on top of the list to get people to see more than just the top ideas and find ideas they might not know about.

@marc1706 Excellent idea to prevent vote hogging, by users who just check the top entries (and thus only vote for the top ones, who already have many votes), which may end up with excellent (but more recent) ideas never getting the attention they deserve. 

"Card" view to show just the essentials of X idea, would be a nice idea. Pretty much the basic design idea by all the forums using widgets to display "recent topics/images/downloads/guides/articles etc" in a small card view at the top of the index forum page.
